src/components/Breadcrumbs/styles.module.css

```diff
@@ -14,6 +14,20 @@
 }
 
 .locBreadcrumbContainer {
-  padding: 16px 0;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 16px 0;
   font-size: 12px;
 }
+
+.search {
+  width: 33%;
+}
+
+@media (max-width: 720px) {
+  body .search{
```

Comment:
i don't think you need the word `body` here

Reply:
can we also add `margin: 16px 16px 0 16px;`

Reply:
It does still work without 'body', so I removed that and added the margin